packages/ui/src/documents/__tests__/MintNftForm.test.tsx

```diff
@@ -59,7 +59,7 @@ describe('Mint NFT Form', () => {
     );
```

Comment:
I don't think you need to pass the user here anymore

Reply:
👍